pmlc/conversion/comp_to_llvm/common.cc

```diff
@@ -112,9 +114,57 @@ getManglingTypes(mlir::MLIRContext *context) {
       mlir::FloatType::getF32(context),    mlir::FloatType::getF64(context)};
 }
 
+namespace {
+struct ConvertFuncSignature : mlir::OpConversionPattern<mlir::FuncOp> {
+  ConvertFuncSignature(mlir::TypeConverter &typeConverter,
+                       mlir::MLIRContext *context)
+      : mlir::OpConversionPattern<mlir::FuncOp>(typeConverter, context) {}
```

Comment:
You can use `using mlir::OpConversionPattern<mlir::FuncOp>::OpConversionPattern;` to pick up the constructors directly.

Reply:
This pattern requires type converter, so I don't want to also pick up constructor that sets it to nullptr.